### ISSUE TITLE
[BUG] Unity WEBGL problem with multiple Live queries.

### ISSUE THREAD

**Issue by dlunhor** [MoralisWeb3/unity-web3-game-kit#108]
I have code in unity with this behaviour. After player subscription initiated, next chat initiation start. All ok with this. Without errors. Also all works in editor and standalone. But I found some mistakes in code. It's not working normally. See screenshots for network pannel.

It is SHOW STOPPER.

Player code
```
MoralisLiveQueryCallbacks<HubPlayer> callbacks = new MoralisLiveQueryCallbacks<HubPlayer>();
MoralisLiveQueryController.AddSubscription(HubMultiplayerKey, query, callbacks);
```

Chat:
```
MoralisLiveQueryCallbacks<TextMessage> callbacks = new MoralisLiveQueryCallbacks<TextMessage>();
MoralisQuery<TextMessage> textQuery = await Moralis.Query<TextMessage>();
MoralisLiveQueryController.AddSubscription(TextMessageKey, textQuery, callbacks);
```

Screnshots:
![chrome_l6VR84ZUyd](https://user-images.githubusercontent.com/8328838/173194295-51c2eea2-56f5-4f77-af59-a9d3b203b23f.png)
![chrome_03QOVzxxrd](https://user-images.githubusercontent.com/8328838/173194317-6683e64f-75aa-4211-9e4e-0baa68cd0f48.png)

**Comment by GilbertoBitt** [comment]
you are not the only one. I'm not even able to transfer BUSD on the finance test. and every time a see a different error the main one been about newton json.

**Comment by xactant** [comment]
@dlunhor Thanks for reporting this. I understand you are having issues with multiple live queries in WebGL. I will look into this.

You mention `But I found some mistakes in code.` can you elaborate on this? Thank you

**Comment by xactant** [comment]
@GilbertoBitt your comment does not seem to fit the topic - did you mean to comment about Live Queries? If so please add more detail about a specific issue. Thank you.

**Comment by dlunhor** [comment]
> @dlunhor Thanks for reporting this. I understand you are having issues with multiple live queries in WebGL. I will look into this.
> 
> You mention `But I found some mistakes in code.` can you elaborate on this? Thank you

I dont understand what part of code has mistake. But alghoritm is broken in this part:
1. Create new socket
2. Binding this socket to unity by id (!)//now id binding is broken
3. In update loop we want to subscribe live query on our socket. But we will write in next socket.
4. Second subscription, don't find any socket and do anything.

**Comment by xactant** [comment]
I was able to replicate this. Built for desktop with 2 live queries. Desktop build Callbacks for both were triggered as expected.

Built for WebGL. I see connection established for both with separate Client IDs however I only observe one subscription established.

Will look into this more ...

**Comment by dlunhor** [comment]
> 

Yep. Very straightforward to replicate. And it's always the same. Problem with binding id's of stream between Unity and JS.

**Comment by xactant** [comment]
Issue fix has been merged in https://github.com/MoralisWeb3/web3-unity-sdk and will be included in the next release.

**Comment by dlunhor** [comment]
> 

When release? Can I have some hotfix?

**Comment by xactant** [comment]
As I mentioned above this is in the main branch. You can download it from  https://github.com/MoralisWeb3/web3-unity-sdk 

We are delayed in the release due to a different issue discovered while testing the release. We should be deploying later this week.

**Comment by KoenRijpstra** [comment]
This is fixed in the latest 1.2.3 release! 🎉
